Library/Homebrew/tab.rb

```diff
@@ -341,6 +341,20 @@ def write
     tabfile.atomic_write(to_json)
   end
 
+  # Helper method for checking truthiness of a value that may not be specified.
+  # `Tab.might_be?(:foo)` differs from `tab.foo` in that if `foo` is not
+  # specified `#might_be?` will return true.
+  def might_be?(key)
```

Comment:
@maxim-belkin when you say "neither", do you mean `might_be?` and `key_truthy_or_absent?`, or `might_be?` and `definitely_not?`?